frontend/src/lib/components/ActivityLog/ActivityLog.tsx

```diff
@@ -4,9 +4,9 @@ import { TZLabel } from 'lib/components/TimezoneAware'
 import { useValues } from 'kea'
 import './ActivityLog.scss'
 import { activityLogLogic } from 'lib/components/ActivityLog/activityLogLogic'
-import { Skeleton } from 'antd'
 import { ActivityScope, Describer } from 'lib/components/ActivityLog/humanizeActivity'
 import { PaginationControl, usePagination } from 'lib/components/PaginationControl'
+import { Skeleton } from '../Skeleton/Skeleton'
```

Comment:
```
import { Skeleton } from '../Skeleton/Skeleton'
Skeleton.Circle
Skeleton
```

That's a lot of `Skeleton` 🤣 

Might be nicer as `SkeletonRow` `SkeletonCircle` etc. 

A little nitty :)

Reply:
I quite like the `Component.Variant` grouping actually

Reply:
I'm also a fan personally. I did the same pattern with `LemonModal`. I don't have a strong reasoning for why I prefer it though 😅

Reply:
I don't have a strong feeling. I wonder if I'd have spotted it if I wasn't in nit-picking mode :)

If it bothers me in usage enough to PR a replacement then we can let future us decide